src/modules/User/resolvers.ts

```diff
@@ -0,0 +1,38 @@
+import { queryField, mutationField, nonNull } from 'nexus/dist';
+import { UserInterface } from './model';
+
+export const me = queryField('me', {
+  type: 'User',
+  resolve: (_, __, ctx) => {
+    if (!ctx.authVerificationResult.userId) {
+      throw new Error('Unexpected middleware behaviour.');
+    }
```

Comment:
Please answer my initial questions https://github.com/makerdao-ses/switchboard-boilerplate/pull/16#discussion_r1135912953 or in other way explain why this unnecessary check is needed? Why you don't need to check if `ctx.prisma.user` exists before accessing it?

Reply:
https://github.com/makerdao-ses/switchboard-boilerplate/pull/16#discussion_r1136824783

Reply:
> Why you don't need to check if ctx.prisma.user exists before accessing it?

because it has been defined as the part of the context. in `context.ts`